.github/workflows/openqa_fullstack.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Clone openQA
         run: git clone --depth 1 https://github.com/os-autoinst/openQA.git ../openQA
       - name: Install dependencies
-        run: (cd ../openQA; bash -x tools/ci/build_cache.sh)
+        run: (cd ../openQA; bash -x tools/ci/build_cache.sh; sudo npm install)
```

Comment:
```suggestion
        run: (cd ../openQA; bash -x tools/ci/build_cache.sh; npm install)
```

Reply:
That didn't work:
https://github.com/perlpunk/os-autoinst/actions/runs/7962626187/job/21736504240#step:6:807
```
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /github/home/.npm
npm ERR! errno -13
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! previous versions of npm which has since been addressed.
npm ERR! 
npm ERR! To permanently fix this problem, please run:
npm ERR!   sudo chown -R 1000:100 "/github/home/.npm"
```